### PR TITLE
fix(TDP-11463): override comparison filters for runtime TQL-DSEL translation

### PR DESCRIPTION
**Pull request by ngrange-talend**
**What is the problem this Pull Request is trying to solve?**
 - DSEL can compare everything. Then `x>1` is true

**What is the chosen solution to this problem?**
- override visit method
- add a matches() filter to be sure this is a number  

**Link to the JIRA issue**
[TDP-11463](https://jira.talendforge.org/browse/TDP-11463)
 
**Please check if the Pull Request fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/Talend/daikon/blob/master/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

File: daikon-tql/daikon-tql-dsel/src/test/java/TqlToDselConverterTest.java [hunk 1]
```diff
@@ -33,646 +36,701 @@ static void setUp() {
...
+                testComparisonOperator(tqlQuery, "id", "1", ELNodeType.LOWER_OR_EQUAL);
+            }
+
+            @Test
+            public void greaterOrEqualThan() {
+                final String tqlQuery = "id >= 1";
+                testComparisonOperator(tqlQuery, "id", "1", ELNodeType.GREATER_OR_EQUAL);
+            }
+        }
     }
```
Comment (ngrange-talend) [inline daikon-tql/daikon-tql-dsel/src/test/java/TqlToDselConverterTest.java:168]: Only this class was added

Comment (fway87) [reply daikon-tql/daikon-tql-dsel/src/test/java/TqlToDselConverterTest.java:168]: Nice !
Could you add an assert on the string representation of the DSEL ELNodes actual and expected ?
You can see more in the assertELNodesAreEquals() method

Comment (ngrange-talend) [reply daikon-tql/daikon-tql-dsel/src/test/java/TqlToDselConverterTest.java]: updated ;)

**Review by fway87 — APPROVED** [review]
It's OK for me

**Pull request merged**